planner/core/testdata/enforce_mpp_suite_in.json

```diff
@@ -6,51 +6,51 @@
       "select @@tidb_enforce_mpp",
       "select @@tidb_opt_tiflash_concurrency_factor",
       "set @@tidb_allow_mpp=0",
-      "explain format='verbose' select count(*) from t where a=1",
-      "explain format='verbose' select /*+ read_from_storage(tikv[t]) */ count(*) from t where a=1",
-      "explain format='verbose' select /*+ read_from_storage(tiflash[t]) */ count(*) from t where a=1",
+      "explain format = 'brief' select count(*) from t where a=1",
```

Comment:
@LittleFall can I change this? because it binds the id of each operator, which will be changed in future due to other PRs.

Reply:
I set `format='verbose'` to check that the task cost is reduced by `enforce_mpp`. So I suggest keeping `verbose` in the test suite `TestEnforceMPP`, and use `format='brief'` to check other tests.